client/MentorPanel/src/redux/actions.js

```diff
@@ -10,10 +10,77 @@ export const QUESTION_ANSWERED = 'QUESTION_ANSWERED'  // question was answered b
 export const QUESTION_ERROR = 'QUESTION_ERROR'        // question could not be answered by mentor
 export const ANSWER_FINISHED = 'ANSWER_FINISHED'      // mentor video has finished playing
 
-export const loadMentor = mentor => ({
-  type: MENTOR_LOADED,
-  mentor: mentor,
-})
+export const MENTOR_TOPIC_QUESTIONS_LOADED = 'MENTOR_TOPIC_QUESTIONS_LOADED'
+export const TOPIC_SELECTED = 'TOPIC_SELECTED'
+
+export const loadMentor = mentor => (dispatch) => {
+  dispatch({
+    type: MENTOR_LOADED,
+    mentor: mentor,
+  })
+  dispatch(loadQuestions(mentor.id))
+}
+
+export const loadQuestions = mentor_id => (dispatch) => {
+  const Papa = require('papaparse/papaparse.min.js')
+  const questions_url = questionsUrl(mentor_id)
+  const questions = {}
+
+  Papa.parse(questions_url, {
+    download: true,
+    complete: (results) => {
+      for (let i = 1; i < results.data.length; i++) {
```

Comment:
It looks like `const questions` should be defined inside this callback (since it has no meaning outside the callback scope?)

As an exercise though, I think it would be good to rewrite this for loop as a `reduce` operation. It would look something like this:

```
const questions = results.data.reduce((acc, cur, i) => {
...
}, {})
```